downstream/modules/platform/ref-guidelines-hosts-groups.adoc

```diff
@@ -20,8 +20,8 @@ If you use one value in `[database]` and both {Controllername} and {HubNameMain}
 ====
 
 .{ControllerNameStart}
-* {ControllerNameStart} does not configure replication or failover for the database that it uses. 
-{ControllerName} works with any replication that you have.
+* {ControllerNameStart} will not configure replication or failover for the database that it uses.
```

Comment:
Hey. Have you considered returning the future tense (will) to present tense (does), which is preferred unless it can't be done?